src/celeritas/ext/detail/G4VecgeomConverter.hh

```diff
@@ -0,0 +1,147 @@
+//----------------------------------*-C++-*----------------------------------//
+// SPDX-FileCopyrightText: 2020 CERN
+// SPDX-License-Identifier: Apache-2.0
+//---------------------------------------------------------------------------//
+/*!
+ * \file G4VecgeomConverter.hh
+ * \brief Class to create a VecGeom model from a pre-existing Geant4 geometry
+ *
+ * Original code from G4VecGeomNav package by John Apostolakis et.al.
+ *
+ * Original source:
+ * https://gitlab.cern.ch/VecGeom/g4vecgeomnav/-/raw/fdd310842fa71c58b3d99646159ef1993a0366b0/include/G4VecGeomConverter.h
+ */
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <utility>
+#include <vector>
+#include <VecGeom/base/TypeMap.h>
+#include <VecGeom/management/GeoManager.h>
+
+#include "Geant4/G4PVReplica.hh"
+#include "Geant4/G4RotationMatrix.hh"
+#include "Geant4/G4ThreeVector.hh"
+
+class G4LogicalVolume;
+class G4AffineTransformation;
+class G4VPhysicalVolume;
+class G4VSolid;
+
+namespace celeritas
+{
+inline namespace VECGEOM_IMPL_NAMESPACE
```

Comment:
Should there be a vecgeom namespace inside of the celeritas namespace?

Reply:
I have moved the few conversion-related classes into the celeritas namespace.